examples/odata-cli/package.json

```diff
@@ -6,7 +6,7 @@
     "private": true,
     "main": "dist/index.js",
     "scripts": {
-        "start": "node -r dotenv/config -r ts-node/register src/index.ts dotenv_config_path=./.env/env.properties",
+        "start": "node -r dotenv/config dist/index.js dotenv_config_path=./.env/env.properties",
```

Comment:
I'm probably in the minority here but I liked the old way, adding a mandatory build step adds a bit of friction in an example

Reply:
No, you are not in the minority but we removed `ts-node` from the devDependency for a reason (I forgot) so, I pretty much have no other option here. Since, it is "just" an example, I think we are fine.